.travis.yml

```diff
@@ -92,11 +100,28 @@ install:
 script:
   - cd ${TRAVIS_BUILD_DIR}
   - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then ulimit -S -n 2048; fi
-  - echo $MAIN_CMD $SETUP_CMD
-  - eval $MAIN_CMD $SETUP_CMD
+  - |
+    if [[ $NAME == 'minimal' ]] || [[ $NAME == 'full' ]] || [[ $NAME == 'Lint' ]] ||  [[ $NAME == 'osx' ]] ||  [[ $NAME == 'old numpy' ]] ||  [[ $NAME == 'numpy dev' ]]; then \
```

Comment:
The "lint" build should not be in this list. It has a completely different `MAIN_CMD`. Also, I am not a huge fan of having `ifs` like that as it breaks the very generic way we had before, I would prefer all this gymnastic to be in the `MAIN_CMD` variable. But if it has to be this way (having everything in `MAIN_CMD` has its issues) I would rather test for the exceptions than for the general case: I would test for `[[ $NAME != 'Doc' ]] && [[ $NAME != 'Lint' ]]` for the first batch and `[[ $NAME == 'Doc' ]] || [[ $NAME == 'Lint' ]]` for the second batch.